<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGeneMainWindow</class>
 <widget class="QMainWindow" name="NewGeneMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NewGene - K-adic data management for the social sciences</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="NewGeneTabWidget" name="tabWidgetMain">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tabOutput">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <attribute name="title">
        <string> Create output dataset </string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayoutOutput">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="NewGeneCreateOutput" name="CreateOutputPane" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabInput">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <attribute name="title">
        <string> Manage input</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayoutOutput">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="NewGeneManageInput" name="ManageInputPane" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInput">
    <property name="title">
     <string>Input</string>
    </property>
    <addaction name="actionNew_Input_Dataset"/>
    <addaction name="actionOpen_Input_Dataset"/>
    <addaction name="actionSave_Input_Dataset_As"/>
    <addaction name="actionClose_Current_Input_Dataset"/>
   </widget>
   <widget class="QMenu" name="menuOutput">
    <property name="title">
     <string>Output</string>
    </property>
    <addaction name="actionNew_Output_Dataset"/>
    <addaction name="actionOpen_Output_Dataset"/>
    <addaction name="actionSave_Output_Dataset_As"/>
    <addaction name="actionClose_Current_Output_Dataset"/>
   </widget>
   <addaction name="menuInput"/>
   <addaction name="menuOutput"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Input_Dataset">
   <property name="text">
    <string>Open Input Dataset...</string>
   </property>
  </action>
  <action name="actionOpen_Output_Dataset">
   <property name="text">
    <string>Open Output Dataset...</string>
   </property>
  </action>
  <action name="actionClose_Current_Input_Dataset">
   <property name="text">
    <string>Close Current Input Dataset</string>
   </property>
  </action>
  <action name="actionClose_Current_Output_Dataset">
   <property name="text">
    <string>Close Current Output Dataset</string>
   </property>
  </action>
  <action name="actionNew_Input_Dataset">
   <property name="text">
    <string>New Input Dataset...</string>
   </property>
  </action>
  <action name="actionSave_Input_Dataset_As">
   <property name="text">
    <string>Save Input Dataset As...</string>
   </property>
  </action>
  <action name="actionNew_Output_Dataset">
   <property name="text">
    <string>New Output Dataset...</string>
   </property>
  </action>
  <action name="actionSave_Output_Dataset_As">
   <property name="text">
    <string>Save Output Dataset As...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>NewGeneTabWidget</class>
   <extends>QTabWidget</extends>
   <header location="global">newgenetabwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NewGeneCreateOutput</class>
   <extends>QWidget</extends>
   <header location="global">newgenecreateoutput.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NewGeneManageInput</class>
   <extends>QWidget</extends>
   <header location="global">newgenemanageinput.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
